### ISSUE TITLE
[Feature] onErrorRetryCountSatisfied

### ISSUE THREAD

**Issue by rostero1** [vercel/swr#288]
The addition of `errorRetryCount` is great.  Would it be possible to get notified whenever swr is no longer retrying?  Maybe through the addition of something like `onErrorRetryCountSatisfied`?

**Comment by rostero1** [comment]
Also, a bit of a tangent, but I'm looking at config's `onErrorRetry`: 

```
  if (!isDocumentVisible()) {
    // if it's hidden, stop
    // it will auto revalidate when focus
    return
  }
```

Is this true if `revalidateOnFocus: false`?

**Comment by huozhi** [comment]
`isDocumentVisible` returns true when page is not visible. It's ok to customize your own `onErrorRetry`

```js
<SWRConfig 
  value={{
     onErrorRetry(err, key, config, revalidate, opts) {
       // count is reached, or you can make your own condition
       if (
         typeof config.errorRetryCount === 'number' &&
         opts.retryCount > config.errorRetryCount
       ) {
         // get notified
       }
       SWRConfig.default.onErrorRetry(err, key, config, revalidate, opts)
     }
  }}
>
  <App />
</SWRConfig>
```

This workaround is available in swr@0.5.3
